### DISCUSSION TITLE
Scalene within docker

### DISCUSSION THREAD

**Discussion by jey07** [plasma-umass/scalene#612] [General]
Is it possible to use scalene within docker. 
I followed the instruction and ran script with scalene and no html file is generated.

**Comment by Exctues** [comment]
Have you tried to use it with `--cli --html` ? I gues  it will print results into stdout by default

**Reply by jey07** [reply]
it will but along with it there will lot of  html codes.

**Reply by bahcruz** [reply]
I set my program to exit after running the code I am profiling.  Then in the docker file I use scalene as ENTRYPOINT
ENTRYPOINT ["scalene", "--html", "--outfile", "sca_profile.html", "foo.py"]
then after program exit, check the Files tab and save html file for later viewing.
